tests/store.yml

```diff
@@ -14,8 +14,24 @@ parameters:
     store:
       enabled: true
       name: thanos-store-1
+    store_alerts:
+      enabled:
+        "*": true
+    receive:
+      enabled: true
+    receive_alerts:
+      enabled:
+        "*": true
+    compactor:
+      enabled: true
+    compactor_alerts:
+      enabled:
+        "*": true
     query:
-      enabled: false
+      enabled: true
+    query_alerts:
+      enabled:
+        "*": true
```

Comment:
Having this config in `tests/store.yml` doesn't really make sense. I'd vote to move the changes either to `tests/defaults.yml` or adding a separate golden test case which enables all alerts.

Also since we're not the only users of the component, please make sure that the component defaults regarding which alerts are enabled don't become a breaking change for other users.

Reply:
> separate golden test case which enables all alerts

Ok, I'll do that.

> Also since we're not the only users of the component, please make sure that the component defaults regarding which alerts are enabled don't become a breaking change for other users.

That's why I enabled no alerts by default. There were none before.